### PR TITLE
Added info panels for planet markers

### PR DESCRIPTION
**Pull request by Jolly5**
Title says it all, again. A Screenshot is in the P-Seminar group.

### CODE REVIEW

File: app/src/main/java/mainburg/planetenweg/LocationMarker.java [hunk 1]
```diff
@@ -165,6 +162,7 @@ public void onLocationChanged(Location location) {
         position.setIcon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_GREEN));
 
         if (trackLocation) {
+            trackLocation = false;// I decided to make it track you the first time your position was determined.
```
Comment (torkleyy) [inline app/src/main/java/mainburg/planetenweg/LocationMarker.java:177]: What happens on screen rotation?

**Comment by torkleyy** [comment]
This is awesome :shipit:

**Comment by torkleyy** [comment]
You gonna make any more commits or shall I merge?

**Comment by Jolly5** [comment]
You can merge this one. I might make an update on the rotation problem tomorrow, but I can create a new PR then.

File: app/src/main/java/mainburg/planetenweg/LocationMarker.java [hunk 2]
```diff
@@ -47,20 +47,32 @@
      * It is needed to notify the user when their location could not have been determined.
      */
     private boolean locationFound;
+
+    private static final String TRACK_LOCATION_KEY = "trackLocation";
     private boolean trackLocation;
+
+    private static final String GPS_DISABLED_KEY = "gpsDisabled";
```
Comment (torkleyy) [inline app/src/main/java/mainburg/planetenweg/LocationMarker.java]: These keys should be prefixed with the package name

**Comment by Jolly5** [comment]
You can still merge.
I only fixed the problem regarding the screen rotation.

**Review by torkleyy — COMMENTED** [review]
Thanks, I have just one concern before we proceed.

**Comment by Jolly5** [comment]
There you go ;)

**Comment by torkleyy** [comment]
Thank you ;)

**Pull request merged**